src/Disp.js

```diff
@@ -1249,6 +1249,56 @@ CM.Disp.ReplaceTooltipLump = function() {
 	}
 };
 
+/**
+ * This function replaces the original Game.crate and Game.crateTooltip functions of stats page
+ */
+CM.Disp.ReplaceCrateTooltipAchievements = function() {
+	let options = {normal: false, shadow: false};
+	switch (CM.ConfigData.MissingAchievements) {
```

Comment:
Can't you just use if-statements? I don't really see the benefit of this.
For example, for `options.normal` could be replaced with `if (CM.ConfigData.MissingAchievements == 1 || (CM.ConfigData.MissingAchievements == 3)`, if you add the option to turn off the display as discussed in the other comment.